KMNavigationBarTransition/UINavigationController+KMNavigationBarTransition.m

```diff
@@ -49,6 +49,10 @@ + (void)load {
     });
 }
 
+- (UIColor *)km_containerViewBackgroundColor {
```

Comment:
No isn't because you can have several subclass of UINavigationController and in each one you would use different backgrohnd color additionaly you could always in subclass choose color from view controller on the top in this case